### PR TITLE
351 pjg reorg demo docs

### PR DESCRIPTION
**Pull request by gleckler1**

### CODE REVIEW

**Comment by durack1** [comment]
@gleckler1 just wondering whether you want to keep any of the `*.png` files - it seems you've purged these for `GFDL` but not for `CSIRO`

**Comment by gleckler1** [comment]
@durack1 @doutriaux1  good to go here... someone please push me...

**Comment by durack1** [comment]
@gleckler1 it looks like you are [failing all 8 of the travis tests](https://travis-ci.org/PCMDI/pcmdi_metrics/builds/132990491#L964) it'd be good to solve that and get the travis test passing before merging into master

@doutriaux1 I think this will require some assistance from you..

**Comment by durack1** [comment]
@gleckler1 you still have the CSIRO `*.png` files in this PR..

**Comment by gleckler1** [comment]
@durack1 @doutriaux1 its good to push ... ok to have one png in there.  Travis text fixed in another branch by CD by removing disclaimer from test.  Also CD is going to cp jsons to test space so as they are updated it does not break test suite.

**Comment by durack1** [comment]
@gleckler1 - 2:
`demo/external-setups/CSIRO/test_4seasRMS.png` and
`demo/external-setups/CSIRO/test_3obsRMS.png`

If you're going to leave some `pngs` in there may as well leave the lot so the examples are complete in my opinion

File: demo/parameter_files/pmp_input_parameters_demo2_save_interpolated_clims.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+##########################################################
+
+
+## FIRST USE OF A PYTHON LIST, IN THIS CASE IT HAS ONLY ONE ENTRY 
+model_versions = ['ACCESS1-0']   # THIS IS A MANDETORY ENTRY FOR DOCUMENTING RESULTS
+
+###############################################################################
+## DATA LOCATION: MODELS, OBS AND METRICS OUTPUT
+## ROOT PATH FOR MODELS CLIMATOLOGIES
+#mod_data_path = '/work/metricspackage/mod_clims/cmip5-amip'
```
Comment (doutriaux1) [inline demo/parameter_files/pmp_input_parameters_demo2_save_interpolated_clims.py]: i would remove the commented out path line

**Comment by durack1** [comment]
@gleckler1 https://github.com/PCMDI/pcmdi_metrics/pull/353 is now merged, so if you tweak and push this again it'll trigger another travis test and if it all passes - merge-aroony..

**Comment by doutriaux1** [comment]
@durack1 there's a bug in the test suite... I just realized it passed no matter what now...

**Comment by durack1** [comment]
@doutriaux1 great.. Well I'm glad you're onto it!

**Comment by doutriaux1** [comment]
my bad it works there, i just broke in my local branch, sorry for the noise

**Comment by durack1** [comment]
@doutriaux1 yes I was a bit curious as [travis reports 8 tests pass](https://travis-ci.org/PCMDI/pcmdi_metrics/builds/133183199#L913-L918)

@gleckler1 so back to my original comment..

**Comment by gleckler1** [comment]
@durack1 I'm ok with only csiro pngs...

**Comment by durack1** [comment]
@gleckler1 so why delete the GFDL pngs then?

**Comment by gleckler1** [comment]
@durack1 @doutriaux1 still waitin...

**Comment by doutriaux1** [comment]
push failed so I restarted the build on travis to be sure why it failed.

**Comment by doutriaux1** [comment]
@gleckler1 you had not merged master in. Rerunning test suite locally with master merged in

**Comment by doutriaux1** [comment]
@durack1 mister keep-it-neat-and-clean please delete branches after merging. Thanks

**Pull request merged**